manual/development-process/production-deployment.md

```diff
@@ -0,0 +1,175 @@
+
+We should not have to hold off production deployments because of user testing. If a feature needs to be tested by users
+before it is available on production, then it should be put behind a [feature toggle] so that the deployment and the
+feature release are independent.
+
+### Step 4: Check what will be deployed to Staging and Production
+
+Every main branch build is tagged with a version number. When we deploy to an AWS environment, the deployment job
+updates the relevant release branch so that it matches the deployed version. See the [architecture decision record on
+versioning][adr-2] for more details on how this works.
+
+Inspect the git history to check the currently deployed versions. For example, if you fetch the latest versio and run
+`git log --graph --pretty=oneline --abbrev-commit`, you should see something like this:
+
+![A git log showing v18 deployed to Staging and Production, and v19 deployed to Integration](./images/git-deployment-versions.png)
+
+This shows that v19 is currently deployed to Integration, and v18 is deployed to Staging and Production.
+
+You can find all the non-merge commits between two release branches by running the command:
+
+```
+git log --pretty=format:"%h%x09%<(20)%an%ad%x09%s" origin/release-staging..origin/release-intg | grep -v "Merge branch" | grep -v "Merge pull request"
+```
+
+If there are commits by anyone else **ask them** before running any further deployments in case there is a good reason
+not to deploy their code to staging and production.
+
+Make a note of the version that contains the commits you want to deploy.
+
+### Step 5: Deploy to Staging
+
+Go to the [production Jenkins instance][tdr-jenkins-prod] and find the deployment job for the relevant repo.
+
+Click 'Build with parameters', and set the `STAGE` parameter to `staging` and the `TO_DEPLOY` parameter to the version
+you found in the last step.
+
+### Step 6: Check the deployment on Staging
+
+As for step 3, wait for the [end-to-end tests][e2e-tests-staging] to succeed, and check you're happy with the change on
+Staging. Again, look out for alerts in the release Slack channel, and check any relevant graphs or logs.
+
+### Step 7: Deploy to Production
+
+You should deploy to Production as soon as possible after deploying to Staging. If the environments are in sync, it
+makes it easier to roll out other changes and be confident that everything works. If you delay the Production
+deployment, a later deployment might succeed on Staging but fail on Production because it was dependent on your change,
+which hadn't been deployed yet.
+
+Run the same deployment job that you ran to deploy to Staging, but set the `STAGE` parameter to `production`.
+
+### Step 8: Check the deployment on Production
+
+Production does not have end-to-end tests, and we don't normally log into Production ourselves to avoid generating test
+consignments in Prod. There are more monitoring alerts set up on Production, such as [file check failures], [load
```

Comment:
"Prod" should be out in full, ie Production

Reply:
Fixed